xpdacq/simulation.py

```diff
@@ -190,6 +180,7 @@ def insert_imgs(mds, fs, n, shape, save_dir=tempfile.mkdtemp()):
 
 # instantiate simulation objects
 db = build_pymongo_backed_broker()
-pe1c = SimulatedPE1C('pe1c', {'pe1_image': lambda: 5})
+db.fs.register_handler('RWFS_NPY', be.ReaderWithFSHandler)
+pe1c = SimulatedPE1C('pe1c', {'pe1_image': lambda: np.ones((5,5))}, fs=db.fs)
```

Comment:
There is the plan for fully integrated with ``xpdSim`` (issue #352) as we discussed a while ago. But I would prefer to do it in separate PR to minimize potential effects.